### PR TITLE
change files name Add-array to transformJson

### PR DESCRIPTION
**Pull request by seb879**
change filename Add_array to transformJson and change constante name addarray to transformJson

### CODE REVIEW

File: functions/transformJson.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const addArray = datas => (
+const transformJson = datas => (
```
Comment (erabin05) [inline functions/transformJson.js]: Which Json ? datas from which entities ?

File: routes/pointofinterest/route.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ router.get('/sample', (req, res) => {
     if (err) {
```
Comment (erabin05) [inline routes/pointofinterest/route.js:14]: It is not in this PR bu you should avoid SELECT * in your SQL request (l. 14)

**Review by erabin05 — CHANGES_REQUESTED** [review]
- when you name a function don't think technical, think which data from which entity you will transform - In you requests you must define the field you want to get an d not all with ```*```

File: functions/transformJson.js (hunk 1 above)
Comment (seb879) [reply functions/transformJson.js]: i have change filename to transformPoiSampleJson

File: routes/pointofinterest/route.js (hunk 2 above)
Comment (seb879) [reply routes/pointofinterest/route.js:14]: done

File: routes/pointofinterest/route.js [hunk 3]
```diff
@@ -2,26 +2,26 @@ const express = require('express');
 
 const router = express.Router();
 const connection = require('../../conf');
-const addArray = require('../../functions/Add_array');
+const transformJson = require('../../functions/transformPoiSampleJson');
```
Comment (erabin05) [inline routes/pointofinterest/route.js]: Keep the same name when you import your function

**Review by erabin05 — CHANGES_REQUESTED** [review]
- keep the same name when you import

File: routes/pointofinterest/route.js (hunk 3 above)
Comment (seb879) [reply routes/pointofinterest/route.js]: corrected

**Review by erabin05 — APPROVED** [review]

**Pull request merged**